kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/control/KmeliaSessionController.java

```diff
@@ -239,6 +240,7 @@ public class KmeliaSessionController extends AbstractComponentSessionController
   // select/deselect all
   private boolean allPublicationsListSelected = false;
 
+  @Inject
```

Comment:
The `@Inject` annotation is useless here because it is dedicated to CDI but here session controllers are constructed programmatically and hence both the `MainSessionController` and `ComponentContext` objects are passed explicitly (not by CDI)